knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml

```diff
@@ -129,6 +129,7 @@ rules:
       - create
```

Comment:
the `IMAGE_KUBE_RBAC_PROXY` is injected via code on the operator, https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/monitoring/rbac_proxy_injection.go

so unlike https://github.com/openshift-knative/eventing-kafka/pull/345/files#diff-9d69b23c98824d912548893af5513af4471530927985e728d1d97aefa712df1cR58 we dont add it here, right?

Reply:
yes correct